config/Config.java

```diff
@@ -0,0 +1,1646 @@
+/*

```

Comment:
It seems that this file is already under `unitex/src/fr/umlv/unitex/config/Config.java`  is there any reason to try to create a new copy under `config/Config.java`?